lib/json_api_resource/queryable.rb

```diff
@@ -25,6 +25,7 @@ def find(id)
       def create(attr = {})
         run_callbacks :create do
           new(:client => self.client_klass.create(attr))
+          self.meta = self.client.last_request_meta
```

Comment:
I'll remove but I guess we never use it, that is why it hasn't busted yet.  We always just .new a thing then .save it